traffic/const_vel_s.py

```diff
@@ -0,0 +1,79 @@
+"""Traffic agent created using just snakeoil interface."""
```

Comment:
Please remove the word "just" and what does the "s" in  const_vel_s, stand for ?